### DISCUSSION TITLE
Date Error

### DISCUSSION THREAD

**Discussion by Wulfheart** [laravel/framework#33527] [General] [answered]
Hello,

sometimes I'm getting a weird error when seeding data:

````
 Doctrine\DBAL\Driver\PDOException::("SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '2020-03-29 02:08:02' for column 'created_at' at row 1")
````

The date itselfs seems fine. I'm also able to insert some thousand entries. Normally it crashes between 2000 and 8000 seeded models. Any way to fix this on MySQL?

**Comment by mfn** [comment]
> The date itselfs seems fine.

However it falls directly into DST, doesn't it? See e.g. https://dba.stackexchange.com/questions/254901/why-cant-i-insert-this-value-using-timestamp-but-can-with-datetime

**Reply by Wulfheart** [reply]
Thanks. But how can I fix this in my specific use case?

> Solution: Just skip that specific time in your table.

Doesn’t seem reasonable

**Comment by Wulfheart** [comment, marked as answer]
Setting the application timezone in ``config/app.php`` worked for me.
